src/components/BookList.js

```diff
@@ -0,0 +1,36 @@
+import PropTypes from 'prop-types';
+import Book from './Book';
+import './BookList.css';
+
+const BookList = ({ books }) => (
+  <div className="main-book-list">
+    <table>
+      <thead>
```

Comment:
- [ ] As the requirements said, BookList should be placed at the `containers` dir. I understand that you created another container component and used this component within. But we need to use the same approach as the instructions told us. Let's use only this component instead, connects with Redux, and move it to the `container` dir
![Screenshot from 2021-05-14 08-42-00](https://user-images.githubusercontent.com/30977845/118265949-4ae8e780-b490-11eb-907d-fa117c3a4068.png)

Reply:
the idea was to keep dispatch functions away from the view.

Reply:
Yes, I get your point. And makes sense.